subsystem/kubeapi_test.go

```diff
@@ -564,6 +564,62 @@ func randomNameSuffix() string {
 	return fmt.Sprintf("-%s", strings.Split(uuid.New().String(), "-")[0])
 }
 
+func printCRs(ctx context.Context, client k8sclient.Client) {
+	if GinkgoT().Failed() {
+		var (
+			err                   error
+			aciList               hiveext.AgentClusterInstallList
+			agentList             v1beta1.AgentList
+			infraEnvList          v1beta1.InfraEnvList
+			bareMetalHostList     bmhv1alpha1.BareMetalHostList
+			nmStateConfigList     v1beta1.NMStateConfigList
+			clusterImageSetList   hivev1.ClusterImageSetList
+			clusterDeploymentList hivev1.ClusterDeploymentList
+		)
+		err = client.List(ctx, &agentList, k8sclient.InNamespace(Options.Namespace))
+		Expect(err).To(BeNil())
```

Comment:
Suggesting not to fail in this function, after all you want to collect everything you can here. 
maybe use multi error to collect all the errors and print them in the end.

Reply:
Done